GeneratorInterface/LHEInterface/scripts/cmsLHEtoEOSManager.py

```diff
@@ -12,15 +12,15 @@
 
 defaultEOSRootPath = '/eos/cms/store/lhe'
 defaultEOSLoadPath = 'root://eoscms/'
-defaultEOSlistCommand = 'xrd eoscms dirlist '
-defaultEOSmkdirCommand = 'xrd eoscms mkdir '
-defaultEOSfeCommand = 'xrd eoscms existfile '
+defaultEOSlistCommand = 'xrdfs eoscms ls '
```

Comment:
`eoscms` will not be recognized universally

the suggestion in #31161 was to use `root://eoscms.cern.ch`, which is a concrete address

Reply:
done